docs/installation/integrations/drupal.md

```diff
@@ -0,0 +1,87 @@
+
+All the collaboration features can also be used for {@link features/collaboration#non-real-time-asynchronous-collaboration asynchronous editing}. It is a collaboration mode when a single actor can work on the document at once, using the revision history, track changes, and comments features to interact with previous and following editors, as the work is done sequentially.
+
+Other features available in the CKeditor&nbsp;5 Premium module include:
+
+* Easily defined {@link features/template templates}.
+* {@link features/slash-commands Slash commands} that let you create, insert, and format rich content on the go.
+* {@link features/format-painter Format painter} to easily style the edited text.
+* Enhanced {@link features/paste-from-office-enhanced Paste from Office} and {@link features/import-word Import from Word} features.
+* Handy one-click {@link features/export-pdf Export to PDF} and {@link features/export-word Export to Word} features that offer portability and cross-platform interoperability.
+* {@link features/mentions Mentions} features that lets you tag other users in comments.
+
+
+## Supported Drupal versions
+
+* Drupal 9 (requires [enabling CKEditor 5](https://www.drupal.org/docs/core-modules-and-themes/core-modules/experimental-ckeditor-5/installation-and-configuration-of-ckeditor-5-module-on-drupal-9))
+* Drupal 10
+
+## Quick start
+
+### Requirements
+
+* PHP 8.0+
+* Drupal 9 with CKEditor 5 enabled
+* Drupal 10
+
+### Installation
+
+Use composer to install the module in Drupal:
+
+```plaintext
+composer require drupal/ckeditor5_premium_features
+```
+
+Alternatively, add the CKEditor 5 Premium Features Module to your Drupal installation.
+
+To do it, enter the **Manage > Extend** section in the Drupal dashboard first and use the **Add new** module button to provide the source for the CKEditor 5 Premium features module. You can fetch the module from https://www.drupal.org/project/ckeditor5_premium_features
+
+{@img assets/img/drupal_dashboard_add_pfm.png 878 Adding CKEditor&nbsp;5 Premium Features module to Drupal.}
+
+### Activation
+
+CKEditor&nbsp;5 Premium module for Drupal requires an active license key. Please follow the {@link support/license-key-and-activation License key and activation} guide to find out how to obtain the key and activate the license. You can also use the {@link @trial guides/overview Premium features Free Trial} to obtain a commitment-free, 30-day trial key.
+
+Once you have the key ready, in the Drupal dashboard go to the CKEditor&nbsp;5 Premium module configuration. In the **General Setting** section, paste the key in the **License key** input box.
+
+{@img assets/img/drupal_general_settings.png 810 Drupal dashboard.}
+
+Next, use the information obtained from the [CKEditor 5 Dashboard](https://dashboard.ckeditor.com/) and fill in the following fields in the Drupal module's **General Settings**.
+
+{@img assets/img/drupal_authorization.png 786 Drupal dashboard.}
```

Comment:
This screenshot doesn't show all required inputs, so it might be confusing that you need to fill out only two from four.

Reply:
It's taken from the Drupal docs, actually, but yes, not full. There is also the configuration part, that we skipped here. We may use the extended version (all input fields) and add the rest of the activation guide here, the thing is do we really want to simple rewrite Drupal docs and struggle to update them in both places every time. Or do we simply trim what we have here (possibly even remove this screenshot) and leave the link to activation/configuration as is? Or do we add the full screenshot and provide a link to "the rest of the activation"?

Reply:
How about splitting the whole screenshot [https://www.drupal.org/files/10_2new.png](https://www.drupal.org/files/10_2new.png) in two parts - one showing the license key input, second showing RTC configuration inputs, and organizing it this way:

> Once you have the key ready, in the Drupal dashboard go to the CKEditor 5 Premium module configuration. In the **General Setting** section, paste the key in the **License key** input box.

\<screen 1 - license key>

> Next, use the information obtained from the [CKEditor 5 Dashboard](https://dashboard.ckeditor.com/) and fill in the following fields in the Drupal module’s **General Settings**.

\<screen 2 - other fields>

Reply:
I see may also use a different screenshot that will make it much more clear, I'll update it in a moment.